### PR TITLE
fix(azure): switch instance types

### PR DESCRIPTION
**Pull request by soyacz**
Due available quotas we switch instance types for Azure tests.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/artifacts/azure-image.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ azure_region_name: "eastus"
 use_preinstalled_scylla: true
 backtrace_decoding: false
 azure_image_db: ''
-azure_instance_type_db: 'Standard_L8s_v2'
+azure_instance_type_db: 'Standard_L8s_v3'
```
Comment (roydahan) [inline test-cases/artifacts/azure-image.yaml:6]: This should be something much smaller.
L2S_V3 is fine.

Comment (soyacz) [reply test-cases/artifacts/azure-image.yaml:6]: this is the smallest

Comment (roydahan) [reply test-cases/artifacts/azure-image.yaml:6]: ok, thx.

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
Checks are failing

**Review by fgelcer — APPROVED** [review]
LGTM

**Comment by fgelcer** [comment]
> Checks are failing

they are failing for unrelated unittest, that are being worked on by @temichus and @fruch

**Comment by fruch** [comment]
> > Checks are failing
> 
> they are failing for unrelated unittest, that are being worked on by @temichus and @fruch

yes, but there one test failing cause of this change:
```
16:24:52      def test_can_create_basic_scylla_instance_definition_from_sct_config():
16:24:52          """Test for azure_region_definition_builder"""
16:24:52          EnvConfig = namedtuple('EnvConfig',
16:24:52                                 ["SCT_CLUSTER_BACKEND", "SCT_TEST_ID", "SCT_CONFIG_FILES", "SCT_AZURE_REGION_NAME",
16:24:52                                  "SCT_N_DB_NODES", "SCT_USER_PREFIX",
16:24:52                                  "SCT_AZURE_IMAGE_DB", "SCT_N_LOADERS", "SCT_N_MONITORS_NODES"])
16:24:52          env_config = EnvConfig(
16:24:52              SCT_CLUSTER_BACKEND="azure",
16:24:52              SCT_TEST_ID="example_test_id",
16:24:52              SCT_CONFIG_FILES=f'["{Path(__file__).parent.absolute()}/azure_default_config.yaml"]',
16:24:52              SCT_AZURE_REGION_NAME="['eastus', 'easteu']",
16:24:52              SCT_N_DB_NODES="3 1",
16:24:52              SCT_USER_PREFIX="unit-test",
16:24:52              SCT_AZURE_IMAGE_DB="/subscriptions/6c268694-47ab-43ab-b306-3c5514bc4112/resourceGroups/scylla-images/providers/"
16:24:52                                 "Microsoft.Compute/images/scylla-5.2.0-dev-x86_64-2022-08-22T04-18-36Z",
16:24:52              SCT_N_LOADERS="2 0",
16:24:52              SCT_N_MONITORS_NODES="1"
16:24:52          )
16:24:52      
16:24:52          os.environ.update(env_config._asdict())
16:24:52          config = SCTConfiguration()
16:24:52          tags = TestConfig.common_tags()
16:24:52          ssh_key = KeyStore().get_gce_ssh_key_pair()
16:24:52          prefix = config.get('user_prefix')
16:24:52          test_config = TestConfig()
16:24:52          builder = region_definition_builder.get_builder(params=config, test_config=test_config)
16:24:52          region_definitions = builder.build_all_region_definitions()
16:24:52      
16:24:52          instance_definition = InstanceDefinition(name=f"{prefix}-db-node-eastus-1", image_id=env_config.SCT_AZURE_IMAGE_DB,
16:24:52                                                   type="Standard_L8s_v2", user_name="scyllaadm", root_disk_size=30,
16:24:52                                                   tags=tags | {"NodeType": "scylla-db", "keep_action": "", 'NodeIndex': '1'},
16:24:52                                                   ssh_key=ssh_key)
16:24:52          assert len(region_definitions) == 2
16:24:52          actual_region_definition = region_definitions[0]
16:24:52      
16:24:52          assert actual_region_definition.test_id == env_config.SCT_TEST_ID
16:24:52          assert actual_region_definition.backend == "azure"
16:24:52          assert actual_region_definition.region == "eastus"
16:24:52          # ignoring user_data in this validation
16:24:52          actual_region_definition.definitions[0].user_data = instance_definition.user_data
16:24:52          # ssh_key is not shown, if actual looks the same as expected possibly ssh_key differ
16:24:52  >       assert instance_definition == actual_region_definition.definitions[0]
16:24:52  E       AssertionError: assert InstanceDefin...ta_disks=None) == InstanceDefin...ta_disks=None)
16:24:52  E         
16:24:52  E         Omitting 9 identical items, use -vv to show
16:24:52  E         Differing attributes:
16:24:52  E         ['type']
16:24:52  E         
16:24:52  E         Drill down into differing attribute type:
16:24:52  E           type: 'Standard_L8s_v2' != 'Standard_L8s_v3'
16:24:52  E           - Standard_L8s_v3
16:24:52  E           ?               ^
16:24:52  E           + Standard_L8s_v2
16:24:52  E           ?               ^
```

**Comment by fgelcer** [comment]
> > > Checks are failing
> > 
> > 
> > they are failing for unrelated unittest, that are being worked on by @temichus and @fruch
> 
> yes, but there one test failing cause of this change:
> 

ugh, missed this one

**Comment by soyacz** [comment]
fixed

**Review by roydahan — APPROVED** [review]

**Pull request merged**